src/js/constants.js

```diff
@@ -47,6 +73,34 @@ export const MediaUIAttributes = {
   MEDIA_LOADING: 'media-loading',
 };
 
+// Maps from state change event type -> attribute name
+export const StateChangeEventToAttributeMap = Object.entries(
+  MediaStateChangeEvents
+).reduce(
+  (mapObj, [key, eventType]) => {
+    const attrName = MediaUIAttributes[key];
+    if (attrName) {
+      mapObj[eventType] = attrName;
+    }
+    return mapObj;
+  },
+  { userinactivechange: 'user-inactive' }
+);
+
+// Maps from attribute name -> state change event type
```

Comment:
Do we need this if we already have it in the const above?

Reply:
Yeah, `'user-inactive'` is the "odd man out" because it isn't one of the `MediaUIAttributes`.